Validation: Duplicate alt text and duplicate description error fixes. (#1420)
dotnet/docs-desktop: dotnet-desktop-guide/framework/winforms/advanced/ellipses-and-arcs-in-gdi.md

---
title: "Ellipses and Arcs in GDI+"

> description: Learn how to draw ellipses and arcs using the System.Drawing.Graphics.DrawEllipse and System.Drawing.Graphics.DrawArc methods.

Reviewer: ```suggestion
description: Learn how to draw ellipses and arcs using the DrawEllipse and DrawArc methods in GDI+ with Windows Forms.
```
